apps/researcher/src/lib/community.ts

```diff
@@ -8,6 +8,7 @@ export interface Community {
   slug: string;
   imageUrl: string;
   createdAt: number;
+  membersCount?: number;
```

Comment:
We also have an enum value called `MembershipCountDesc`. Align the names? E.g. `membershipCount` or `MembersCountDesc` for the enum value? Or something different: `numberOfMembers`?